collections/misc/collpermissions.php

```diff
@@ -354,11 +371,13 @@ function verifyAddRights(f){
 					<?php
 					if((array_key_exists("CollAdmin",$USER_RIGHTS) && in_array($collId,$USER_RIGHTS["CollAdmin"]))){
 						?>
+						<form>
```

Comment:
I'm now seeing TVP not liking the fact that there's no submit button in this form.

Instead of adding a form element here where no input is harvested from the end user, how weird does it look if we change to the `<section class="fieldset-like"...` pattern here? If it looks really different from the other fieldset elements, we might should change them all to the fieldset-like pattern on this page for consistency.